dashboard/src/components/QueryParams/QueryParams.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withRouter } from "react-router-dom";
+
+function expandParams(params, keys) {
+  return Array.from(params).reduce((acc, entry) => {
```

Comment:
`Array.from` doesn't work in any version of IE

Reply:
Good call out, IE11 ruins everything.

Added `core-js/es6/array` which might be overkill but should only ever be pulled down for IE users and means we can safely use `isArray`, `slice`, etc. as well.